scenes/slime.tscn

```diff
@@ -51,9 +51,16 @@ sprite_frames = SubResource("SpriteFrames_4gpd6")
 autoplay = "default"
 frame = 2
 frame_progress = 0.0770326
+flip_h = true
```

Comment:
Je žádoucí tento výchozí stav? Chceme mít slime enemy flipnuté na start hry?

Reply:
Ano, je to žádoucí